### DISCUSSION TITLE
Could we get dislikes on shorts too?

### DISCUSSION THREAD

**Discussion by liwwyy** [Anarios/return-youtube-dislike#1071] [Ideas]
I'm not sure if the plugin provides support for shorts but for me, it does not show them I'd really like to see that feature!
![Untitled](https://github.com/user-attachments/assets/3935e741-bcd7-44e3-b389-109ab9655401)

**Comment by jpa102** [comment]
idk about you, but it displays just fine for me

![image](https://github.com/user-attachments/assets/cce7ed27-8db7-4009-a63d-2a5233c9d1b1)

**Reply by liwwyy** [reply]
This must be a bug then? Yeah this plugin is great might as well do some reaserch on it and stop being lazy

**Reply by jpa102** [reply]
yes, it's indeed a bug considering it stopped showing to me a few hours ago *(i'm using the extension and not the userscript version)*

before that happened, it does display dislikes on shorts too. it just doesn't display it properly (pressing the dislike button makes the count appear until you open the comment section or something)

something probably got updated in the backend and now the word "Dislike" overwrites the injected dislike counts every time *(similar behavior can be seen when pressing the dislike button in a long-form video when signed out)*

> and stop being lazy

idk if you're calling me lazy, but here are 4 screenshots *(i had to use the userscript version for these)*

the half functional "non-rounded" design (action buttons are bugged out)

![スクリーンショット (2617)](https://github.com/user-attachments/assets/62c71404-7866-4664-956d-013b4a4d0026)

visible comment section (dislike count appears with no problems)

![スクリーンショット (2618)](https://github.com/user-attachments/assets/8537313f-4bae-40cd-be4c-967f3ae829f4)

the current rounded design (dislike count appears)

![スクリーンショット (2619)](https://github.com/user-attachments/assets/1cf52635-5d0e-41f4-8e16-2a8edcdf6bc8)

visible comment section (dislike count disappears until you press the dislike button)

![スクリーンショット (2620)](https://github.com/user-attachments/assets/314f6b29-1059-440e-ba59-42607224dbeb)

for the "non-rounded" looking design, it's a result of me [using this userscript](https://github.com/aubymori/YouTubePatchCollection)

that's because i found out that injecting dislike counts in the "non-rounded" design in there is more reliable than in the current design that youtube has today

if the devs see this, they can take notes from this i guess...

**Reply by liwwyy** [reply]
Whoa I really didn't expect a reply for my drunken post here and NO I wasn't calling you lazy I was calling my-self lazy 😅

**Reply by jpa102** [reply]
that's okay, i'm not someone that easily detects sarcasm

this has been an ongoing bug for many months now, and the backend update made it worse (unless you're using the userscript version for the time being)

**Comment by sflesch** [comment]
I just noticed this today. I thought I remembered them working previously. Glad to know I was correct. Hopefully the extension will get updated.
